src/popup/router/pages/TransferSend.vue

```diff
@@ -55,8 +49,8 @@
             <Button
               data-cy="review-withdraw"
               half
-              :disabled="!validAddress || !+form.amount || form.amount <= 0"
-              @click="step = 2"
+              :disabled="!+form.amount || form.amount <= 0"
```

Comment:
Disabled logic now is not the same, it won't be disabled if there are errors in the address input.